plugins/com.google.cloud.tools.eclipse.sdk.ui/src/com/google/cloud/tools/eclipse/sdk/ui/preferences/CloudSdkPreferenceArea.java

```diff
@@ -128,6 +139,31 @@ public void widgetSelected(SelectionEvent e) {
     Dialog.applyDialogFont(contents);
     return contents;
   }
+  
+  private void updateSelectedVersion() {
+    String version = Messages.getString("UnknownVersion"); //$NON-NLS-1$
+    if (!CloudSdkManager.isManagedSdkFeatureEnabled()
+        || (useLocalSdk != null && useLocalSdk.getSelection())) {
```

Comment:
`useLocalSdk` is always `null` if `isManagedSdkFeatureEnabled()` is false (and always non-null otherwise). So, I don't like the null check for testing if our debug flag is on; we should directly check `isManagedSdkFeatureEnabled()` instead.

Reply:
done